src/components/screens/map-screen/map-screen.tsx

```diff
@@ -274,7 +274,21 @@ class MapScreen extends React.Component<Props, State> {
     } catch (error) {}
     this.setState({ loadingUserSettings: false });
   }
+  /**
+   * Adds users saved locations in location from and location in options. 
+   */
 
+   private setLocationOptions = () => {
```

Comment:
```suggestion
   */
   private setLocationOptions = () => {
```
No space needed between Tsdocs and method

Reply:
done